OWNERS

```diff
@@ -3,13 +3,12 @@ filters:
     reviewers:
       - beekhof
       - slintes
-      - n1r1
-      - mshitrit
       - razo7
+      - mshitrit
     approvers:
       - beekhof
       - slintes
-      - n1r1
+      - razo7   
       - mshitrit
 
     # Specify only if your Bugzilla product is not OCP
```

Comment:
Maybe the product should be updated to `Red Hat Workload Availability` ?

Reply:
Not sure if this ever worked. And it seems to be specific to bugzilla. I think you can remove it...